test/sqllogictest/mzcompose.py

```diff
@@ -171,6 +171,7 @@ def compileFastSltConfig() -> SltRunConfig:
         "test/sqllogictest/explain/*.slt",
         "test/sqllogictest/autogenerated/*.slt",
         "test/sqllogictest/transform/*.slt",
+        "test/sqllogictest/release-builds/*.slt",
```

Comment:
Maybe could exclude this directory when running with `--dev`, but we can also pick this up in QA team later.